docs/others/observer.rst

```diff
@@ -72,38 +73,38 @@ Subject
 Subject initialization
 ----------------------
 
-Subjects have to be static or global :c:expr:`lv_subject_t` type variables.
+Subjects have to be static or global :cpp:expr:`lv_subject_t` type variables.
 
-To initialize a subject use :c:expr:`lv_subject_init_<type>(&subject, <params>, init_value)`.
+To initialize a subject use :cpp:expr:`lv_subject_init_<type>(&subject, <params>, init_value)`.
```

Comment:
you cannot have \<params\> in the extression. It only evaluates actual C/CPP code. So this one should be corrected so that the \<params\> is replaced or remove the :cpp:expr: directive

Reply:
replaced to `params`